website/docs/reference/dbt-jinja-functions/target.md

```diff
@@ -17,6 +17,7 @@ Some configs are shared between all adapters, while others are adapter-specific.
 ## Common
 | Variable | Example | Description |
 | -------- | ------- | ----------- |
+| `target.profile_name` | jaffle_shop | The name of the active profile |
```

Comment:
TIL!

Reply:
I was thinking about this a little more.

What does this compile to in dbt Cloud? Can we set it anywhere?

We have some notes above about how to set these values in dbt Cloud, and I have no idea what the answer is for `profile_name` 😅 

If you're curious too, feel free to investigate!